example-server.py

```diff
@@ -74,11 +77,32 @@ def multiply(parent, x, y):
     myfolder = objects.add_folder(idx, "myEmptyFolder")
     myobj = objects.add_object(idx, "NewObject")
     myvar = myobj.add_variable(idx, "MyVariable", 6.7)
+    myvar.set_writable()    # Set MyVariable to be writable by clients
     myarrayvar = myobj.add_variable(idx, "myarrayvar", [6.7, 7.9])
     myarrayvar = myobj.add_variable(idx, "myStronglytTypedVariable", ua.Variant([], ua.VariantType.UInt32))
     myprop = myobj.add_property(idx, "myproperty", "I am a property")
     mymethod = myobj.add_method(idx, "mymethod", func, [ua.VariantType.Int64], [ua.VariantType.Boolean])
-    multiply_node = myobj.add_method(idx, "multiply", multiply, [ua.VariantType.Int64, ua.VariantType.Int64], [ua.VariantType.Int64])
+
+    inargx = ua.Argument()
+    inargx.Name = "x"
+    inargx.DataType = node._guess_uatype(ua.Variant(None, ua.VariantType.Int64))
+    inargx.ValueRank = -1
+    inargx.ArrayDimensions = []
+    inargx.Description = ua.LocalizedText("First number x")
+    inargy = ua.Argument()
+    inargy.Name = "y"
+    inargy.DataType = node._guess_uatype(ua.Variant(None, ua.VariantType.Int64))
+    inargy.ValueRank = -1
+    inargy.ArrayDimensions = []
+    inargy.Description = ua.LocalizedText("Second number y")
+    outarg = ua.Argument()
+    outarg.Name = "Result"
+    outarg.DataType = node._guess_uatype(ua.Variant(None, ua.VariantType.Int64))
+    outarg.ValueRank = -1
+    outarg.ArrayDimensions = []
+    outarg.Description = ua.LocalizedText("Multiplication result")
+
+    multiply_node = myobj.add_method(idx, "multiply", multiply, [inargx, inargy], [outarg])
```

Comment:
I got several complains about the example file being too big, so I do not want to merge such a huge thing. Can you put it in another file? like for example-server-methods.py? Then we just leave the simplest method example in example-server.py?